spec/Error/OutOfOrderCallError_spec.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+describe('OutOfOrderCallError', () => {
+  let OutOfOrderCallError = require('../../src/Error/OutOfOrderCallError.js');
+
+  it('should stringify correctly without args', () => {
+    expect(new OutOfOrderCallError({
+      name: 'foo'
+    }, [], [], []).message).toEqual('Out of order function call foo()');
```

Comment:
Same here, would be nice to see that it generates the rest of the error properly